ingestion_server/ingestion_server/cleanup.py

```diff
@@ -298,6 +299,12 @@ def __init__(self):
         self.s3 = self._get_s3_resource()
         self.s3_bucket = self.s3.Bucket(bucket_name)
         self.date = time.strftime("%Y-%m-%d")
+        self._recreate_local_dir()
+
+    def _recreate_local_dir(self):
+        if os.path.exists(self.local_path):
+            os.system(f"rm -rf {self.local_path}")
+        os.makedirs(self.local_path)
```

Comment:
We should _definitely_ be preferring something like [`pathlib`](https://docs.python.org/3/library/pathlib.html) over using `os` calls and particularly `os.system` directly. If `local_path` is initialized as a `pathlib.Path` object, this becomes:

```suggestion
    def _recreate_local_dir(self):
        self.local_path.unlink(missing_ok=True)
        self.local_path.mkdir()
```

Reply:
Ah! Seems like [`shutil.rmtree`](https://docs.python.org/3/library/shutil.html#shutil.rmtree) might work for that case 🙂

Reply:
Yeah, I applied exactly that ✅